src/app.ts

```diff
@@ -1,7 +1,32 @@
 import { ExpressDriver } from './drivers/drivers';
+import { ServerlessCache } from './cache';
+import * as request from 'request';
+import { reportError } from './shared/SentryConnector';
+const APP_STATUS = process.env.APP_STATUS_URI;
 // ----------------------------------------------------------------------------------
 // Initializations
 // ----------------------------------------------------------------------------------
 
 // ----------------------------------------------------------------------------------
 ExpressDriver.start();
+
+async function fillCache() {
```

Comment:
Can you move this logic into the ServerlessCache module? It will be easier to separate when we move away from using the Gateway.

Reply:
Resolved with 14af224